lms/templates/courseware/progress.html

```diff
@@ -80,7 +80,10 @@ <h2 class="title">${_("We're working on it...")}</h2>
                     <p class="copy">${_("We're creating your certificate. You can keep working in your courses and a link to it will appear here and on your Dashboard when it is ready.")}</p>
                   </div>
                   <div class="msg-actions"></div>
-                  %elif not is_id_verified or is_unverified:
+                  %elif missing_required_verification or is_unverified:
```

Comment:
@adampalay That was a topic @rlucioni and I discussed.
According to what I can still remember from ECOM-4007, if the learner went through ID verification and approved, but they have a generated certificate with `unverified` status, we still do not show the cert gen button. It is a behavior I coded in the PR of ECOM-4007 and we are hesitant to change that behavior in this PR. The thinking is we will not make the logic change here until there are more questions raised. We can update this behavior easily with follow up PR if the logic above needs to be updated.